src/project/qml/MuseScore/Project/internal/ScoresPage/CloudScoresView.qml

```diff
@@ -114,24 +127,276 @@ ScoresView {
+                                accessible.name: modifiedColumn.header + ": " + modifiedLabel.text
+                                accessible.role: MUAccessible.StaticText
+
+                                onActiveChanged: {
+                                    if (active) {
+                                        listItem.scrollIntoView()
+                                    }
+                                }
+                            }
+
+                            anchors.margins: -radius
+                            radius: 2 + border.width
+                        }
+                    }
+                },
+
+                ScoresListView.ColumnItem {
+                    id: sizeColumn
+                    header: qsTrc("project", "Size", "file size")
+
+                    width: function (parentWidth) {
+                        let parentWidthExclusingSpacing = parentWidth - list.columns.length * list.view.columnSpacing;
+                        return 0.13 * parentWidthExclusingSpacing
+                    }
+
+                    delegate: StyledTextLabel {
+                        id: sizeLabel
+                        text: score.fileSize ?? ""
+
+                        font: ui.theme.largeBodyFont
+                        horizontalAlignment: Text.AlignLeft
+
+                        NavigationFocusBorder {
+                            navigationCtrl: NavigationControl {
+                                name: "SizeLabel"
+                                panel: navigationPanel
+                                row: navigationRow
+                                column: navigationColumnStart
+                                enabled: sizeLabel.visible && sizeLabel.enabled && !sizeLabel.isEmpty
+                                accessible.name: sizeColumn.header + ": " + sizeLabel.text
+                                accessible.role: MUAccessible.StaticText
+
+                                onActiveChanged: {
+                                    if (active) {
+                                        listItem.scrollIntoView()
+                                    }
+                                }
+                            }
+
+                            anchors.margins: -radius
+                            radius: 2 + border.width
+                        }
+                    }
+                },
+
+                ScoresListView.ColumnItem {
+                    id: viewsColumn
+
+                    //: Stands for "The number of times this score was viewed on MuseScore.com".
+                    //: Used as the header of this column in the scores list.
+                    header: qsTrc("project", "Views", "number of views")
+
+                    width: function (parentWidth) {
+                        let parentWidthExclusingSpacing = parentWidth - list.columns.length * list.view.columnSpacing;
+                        return Math.max(0.08 * parentWidthExclusingSpacing, 76)
```

Comment:
Please explain exactly why 0.08

Reply:
These numbers are simply percentages, determining the width of the column relative to the parent. The designs don't provide rules for this; so we just use such percentage that it approximately matches the pictures from the design and so that it also looks good on smaller screens. 

So, yes, these are magic numbers, but in the case of UI things like this, those are sometimes justifiable in my opinion.